pkgs/development/tools/misc/asls/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+  , makeWrapper
+  , fetchurl
+  , erlangR22
+}:
+
+stdenv.mkDerivation rec {
+  pname = "asls";
+  version = "0.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/saulecabrera/asls/releases/download/v0.3.0/bin.tar.gz";
+    sha256 = "01d6v79zqw62pkv33km090nyn78zxr0vs3yvnw24ykysvqjyaqd3";
+  };
```

Comment:
Why not build from source?

Reply:
Two main reasons I went with this approach:

- The available tools for the BEAM seem to have several drawbacks not suited for this package:
   - `buildHex`: requires the package to be published in https://hex.pm/ which is not the case for `asls`
   - `buildMix`: requires the package to be a library,` asls `is  an application (BEAM CLI application)
- The other option is to run `mix` commands  during the build phase, that works and that was my initial approach, but that breaks sandboxing (a similar case can be found in this thread: https://discourse.nixos.org/t/question-about-packaging-elixirls/4056)

Until there is a more mature built-in approach I believe this is the most straightforward way.